src/IDelegateRegistry.sol

```diff
@@ -10,40 +10,61 @@ pragma solidity ^0.8.20;
 interface IDelegateRegistry {
     /// @notice Delegation type
     enum DelegationType {
-        NONE,
```

Comment:
Good to populate the zero enum with an empty NONE value, to avoid people passing incorrect default values